### PR TITLE
Add font size utilities.

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Utility classes are the wei. Why?

- more performant than inline styles
- more composable
- less dev mental overhead
- keeps the UI more consistent (limited choice of values for things)

Need more convincing? see https://johnpolacek.github.io/the-case-for-atomic-css/ and https://tailwindcss.com/docs/utility-first#maintainability-concerns

Anyways, this PR:
- adds utility classes for font size. Classes stolen from https://tailwindcss.com/docs/font-size
- replaces any small fontSize inline style (below 1rem) with either `text-sm` or `text-xs`

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/VolumeChart/index.tsx [hunk 1]
```diff
@@ -97,12 +97,11 @@ export default function VolumeChart({
   }, [cv, duration, projectId, showGraph])
 
   const buttonStyle: CSSProperties = {
-    fontSize: '0.7rem',
     textTransform: 'uppercase',
   }
 
   const axisStyle: SVGProps<SVGTextElement> = {
-    fontSize: 11,
+    fontSize: '0.75rem',
```
Comment (johnnyd-eth) [inline src/components/VolumeChart/index.tsx:104]: Why this here? Can the `<text/>` not take a class?

File: src/components/inputs/Pay/PayInputSubText.tsx [hunk 2]
```diff
@@ -125,16 +125,16 @@ export default function PayInputSubText({
     <div
       style={{
         marginTop: '0.3rem',
-        fontSize: '.65rem',
```
Comment (johnnyd-eth) [inline src/components/inputs/Pay/PayInputSubText.tsx:128]: Bumping this one up a bit are we sure?

Comment (johnnyd-eth) [reply src/components/inputs/Pay/PayInputSubText.tsx:128]: Screen Shot 2022-08-25 at 8 19 41 pm

Still looks alright. Up to you

File: src/styles/utilities.scss [hunk 3]
```diff
@@ -44,3 +44,57 @@
 .selected-box-shadow {
   box-shadow: 2px 0px 10px 0px var(--boxShadow-primary)
 }
+
+/* FONT SIZES */
+.text-xs {
```
Comment (benschac) [inline src/styles/utilities.scss:49]: question: Could we do something similar to: https://github.com/jbx-protocol/juice-interface/blob/main/src/hooks/JuiceTheme.ts#L8 and https://github.com/jbx-protocol/juice-interface/blob/main/src/models/semantic-theme/colors.ts#L22



The goal would be to get these constants in but also get some type checking in like colors.

**Comment by tomquirk** [comment]
Closing for now

**Pull request closed without merging**